.circleci/config.yml

```diff
@@ -769,8 +769,25 @@ jobs:
             aws s3 cp ${WHEELFILE} s3://core-previews/${PREVIEW_BRANCH}/ --acl public-read
             cd ../..
 
+            echo -e "Here is the link to download the wheel file: https://core-previews.s3-us-west-2.amazonaws.com/${PREVIEW_BRANCH}/${WHEELFILE}"
             echo -e "https://core-previews.s3-us-west-2.amazonaws.com/${PREVIEW_BRANCH}/${WHEELFILE}" >> S3_URL
 
+      - run:
```

Comment:
Hm, do we even need a separate build stage for this?

I feel like everything that we want to do could be accomplished by adding something like the following to the "Upload wheel to S3" build stage above. (Written in pseudocode because I don't know the correct bash syntax off the top of my head)

```
if branch === "release/demo":
    aws s3 cp ${WHEELFILE} s3://core-previews/${PREVIEW_BRANCH}/streamlit-11.11.11-py2.py3-none-any.whl --acl public-read

```

Reply:
Hm, after thinking about this a bit, I feel like there's no reason to have a separate stage for this and would prefer this given it ends up being a 2-line change unless there's some reason it won't work